Networking/Networking/Model/OrderItemAttribute.swift

```diff
@@ -0,0 +1,33 @@
+public struct OrderItemAttribute: Decodable, Hashable, Equatable {
```

Comment:
nit: Maybe a comment on the top of the struct could help identify what type of entity is this. 👍

Reply:
added here: https://github.com/woocommerce/woocommerce-ios/pull/3085/commits/98fa53c122601a936293eca646fca77fd2da2790